application/testing/CMakeLists.txt

```
@@ -272,10 +272,14 @@ if(VTK_VERSION VERSION_GREATER 9.0.1)
   # No alternative baseline supports in F3D
   if(F3D_MODULE_RAYTRACING)
     f3d_test(NAME TestInteractionCheatsheetRaytracing DATA cow.vtp INTERACTION) #H
+    f3d_test(NAME TestInteractionCheatsheetWhiteBGRaytracing DATA cow.vtp ARGS --bg-color=1,1,1 INTERACTION) #H
+    f3d_test(NAME TestInteractionCheatsheetBlackBGRaytracing DATA cow.vtp ARGS --bg-color=0,0,0 INTERACTION) #H
     f3d_test(NAME TestInteractionCheatsheetScalarsRaytracing DATA dragon.vtu ARGS --scalars --comp=-2 INTERACTION) #HSSS
     f3d_test(NAME TestInteractionNoFileCheatsheetRaytracing INTERACTION NO_DATA_FORCE_RENDER) #HXZM 
-  else()
+    else()
```

revert the identation please
```suggestion
else()
```